Sources/RInAppMessaging/Utilities/BundleInfo.swift

```diff
@@ -32,6 +32,14 @@ internal class BundleInfo {
     class var inAppConfigurationURL: String? {
         return Bundle.main.infoDictionary?[Constants.Info.configurationURLKey] as? String
     }
+    
+    class var customFontNameRegularWeight: String? {
```

Comment:
Do you use `class var` in order to override`customFontNameRegularWeight` variable?
If not, can you use static var?